sdk/boards/ibex-arty-a7-100.json

```diff
@@ -23,6 +23,14 @@
         "gpio_led0" : {
             "start": 0x8f00f000,
             "length": 0x800
+        },
+        "dmb_cfg": {
```

Comment:
What's this?

Reply:
Accidental things from the network stack bits.